runtime/near-vm-logic/src/logic.rs

```diff
@@ -2389,6 +2389,22 @@ impl<'a> VMLogic<'a> {
         Ok(res? as u64)
     }
 
+    /// Debug print given utf-8 string to node log. It's only available in Sandbox node
+    ///
+    /// # Errors
+    ///
+    /// * If string is not UTF-8 returns `BadUtf8`
+    ///
+    /// # Cost
+    ///
+    /// 0
+    #[cfg(feature = "sandbox")]
+    pub fn debug_log(&mut self, len: u64, ptr: u64) -> Result<()> {
```

Comment:
```suggestion
    pub fn sandbox_debug_log(&mut self, len: u64, ptr: u64) -> Result<()> {
```

let's add `sandbox_` prefix to all of the sandbox calls, so the "sandbox-only" aspect of this is also visible at the call-site in the contract.